feathr_project/feathr/definition/sink.py

```diff
@@ -100,4 +111,61 @@ def to_feature_config(self) -> str:
             }
         """)
         hocon_config = tm.render(sink=self)
-        return hocon_config
\ No newline at end of file
+        return hocon_config
+
+    def to_argument(self):
+        return self.output_path
+
+    
+class GenericSink(Sink):
```

Comment:
Add docstring for this. What is the purpose for `GenericSink`?

Reply:
It's the base class of CosmosDbSink, while itself can be used directly, not meant to end users but for other codes, like we plan to add job submission function in the registry.

Reply:
Can we add those info in the code comments for easier maintenance?

Reply:
Comments added